Utilities/build-using-self

```diff
@@ -14,33 +14,46 @@
 set -eu
 
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+root_dir="$(cd ${__dir}/.. && pwd)"
 
-cd "${__dir}/.."
+cd "${root_dir}/"
 echo "Current directory is ${PWD}"
 
 CONFIGURATION=debug
 export SWIFTCI_IS_SELF_HOSTED=1
 
+# Ensure SDKROOT is configured
+host=$(uname)
+if [ "${host}" == "Darwin" ]; then
+    export SDKROOT=$(xcrun --show-sdk-path --sdk macosx)
+fi
+
 set -x
 
+env | sort
```

Comment:
Do we need this? I assume it was for debugging?

Reply:
Primarily informative information.  I think it's valuable to have.